stub/src/main/java/io/grpc/stub/ClientCalls.java

```diff
@@ -288,8 +281,7 @@ private static StatusRuntimeException toStatusRuntimeException(Throwable t) {
   private static RuntimeException cancelThrow(ClientCall<?, ?> call, Throwable t) {
     try {
       call.cancel(null, t);
-    } catch (Throwable e) {
```

Comment:
Assertion can be disabled. So this change may skip logging some error messages, its safer to not change it.

Reply:
The assertion is checking for checked exceptions, which should't happen. Catching Throwable was because this was written before we could write `RuntimeException | Error`. I'm not worried about this change.